products/BellHybrid/apps/application-bell-settings/models/alarm_settings/AlarmSettingsModel.cpp

```diff
@@ -18,20 +18,17 @@ namespace app::bell_settings
         }
     }
 
-    AlarmToneModel::AlarmToneModel(sys::Service *app, SimpleSoundsRepository &soundsRepository)
-        : gui::SettingsModel<UTF8>{app}, soundsRepository{soundsRepository}
+    AlarmToneModel::AlarmToneModel(sys::Service *app) : gui::SettingsModel<UTF8>{app}
```

Comment:
You can completely remove this constructor and instead use `using SettingsModel::SettingsModel;` in `.hpp` file